gms/feature_service.cc

```diff
@@ -43,12 +43,12 @@ constexpr std::string_view features::XXHASH = "XXHASH";
 constexpr std::string_view features::ROLES = "ROLES";
 constexpr std::string_view features::LA_SSTABLE = "LA_SSTABLE_FORMAT";
 constexpr std::string_view features::STREAM_WITH_RPC_STREAM = "STREAM_WITH_RPC_STREAM";
```

Comment:
I wouldn't say the feature is deprecated, in fact it's being made mandatory.

I understand what is meant, but it's better to be more precise.